draft-ietf-ppm-dap.md

```diff
@@ -717,43 +715,29 @@ enum {
 
 The time_interval query type is described in {{time-interval-query}}; the
 fixed_size query type is described in {{fixed-size-query}}. Future
-specifications may introduce new query types as needed (see
-{{query-type-reg}}). Implementations are free to implement only a subset of the
-available query types.
+specifications may introduce new query types as needed (see {{query-type-reg}}).
+Implementations are free to implement only a subset of the available query
+types.
 
 A query includes parameters used by the Aggregators to select a batch of
 reports specific to the given query type. A query is defined as follows:
 
 ~~~
 opaque BatchID[32];
 
-enum {
-  by_batch_id(0),
-  current_batch(1),
-  (255)
-} FixedSizeQueryType;
-
-struct {
-  FixedSizeQueryType query_type;
-  select (FixedSizeQuery.query_type) {
-    case by_batch_id: BatchID batch_id;
```

Comment:
It is OK to remove `by_batch_id` because this fixed-size query type was used only for repeated collection of the same batch with differing aggregation parameters. (Recovering from transient failures in collecting a given batch is enabled via re-querying the same collection job ID.)

Reply:
SGTM.